targets/mbedos5/README.md

```diff
@@ -50,6 +50,15 @@ jshint is used to statically check your JavaScript code, as part of the build pr
 This ensures that pins you are using in your code are available on your chosen target
 platform.
 
+### Python modules
+
+There are Python module dependencies which can be installed with the following command: 
+	`pip install -r requirements.txt`
+Use this command in your .../jerryscript/targets/mbedos5/tools directory. 
+
+If there is no pip installed, use the following command:
+	`apt-get -y install python-pip`
```

Comment:
Same here.

Additionally, this only works if the user is on an apt-based system like Ubuntu. A generic description can be found on pip's homepage: https://pip.pypa.io/en/stable/installing/ Might be worth including it in the description.